planetiler-custommap/README.md

```diff
@@ -0,0 +1,72 @@
+# Configurable Planetiler Schema
+
+It is possible to customize planetiler's output from configuration files.  This is done using the parameter:
+`--schema=schema_file.yml`
+
+The schema file provides information to planetiler about how to construct the tiles and which layers, features, and attributes will be posted to the file.  Schema files are in [YAML](https://yaml.org) format.
+
```

Comment:
Please add a link to an example schema file at the end of the intro paragraph.

Reply:
I would recommend renaming this file to "SCHEMA.md" or so. And then you could add a readme which has the structure 
- what is this good for 
- install and build instructions 
- known limitations / todos

Reply:
At less than 100 lines, I'm not sure we really need separate files to document this?

Reply:
I've added links to the sample schemas and test cases in the opening section, which should resolve the ask for examples.